indexing-hadoop/src/main/java/io/druid/indexer/DetermineHashedPartitionsJob.java

```diff
@@ -102,7 +102,7 @@ public boolean run()
       } else {
         groupByJob.setNumReduceTasks(config.getSegmentGranularIntervals().get().size());
       }
-      JobHelper.setupClasspath(config, groupByJob);
+      JobHelper.setupClasspath(JobHelper.distributedClassPath(config.getWorkingPath()), groupByJob);
```

Comment:
Here, yes. The abstractions were not perfect because some functions (like this JobHelper still took specific configs instead of specific distributed classpaths.